manage.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
No. This is handled by the python environment.

Reply:
I changed it back now.
https://github.com/unioslo/mreg/blob/17eb02178d5f4025b27754ab3ce4def75e06fd8e/manage.py#L1